recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+with_setuptools_wheel:
```

Comment:
For my understanding: What does this do without it being in a zip_keys?

Reply:
This results in two variants (since it is used in the meta.yaml). One is for python <3.13 to keep setuptools and wheel as dependencies. The other is for python >=3.13 to not depend on setuptools and wheel.